interface/weno/import_pharmacies.php

```diff
@@ -0,0 +1,106 @@
+<?php
```

Comment:
For processing, recommend an <I> tag with classes fa fa-spin and fa-circle-o-notch.